src/Interface/Modules/Render/Screenshot.cc

```diff
@@ -49,11 +49,21 @@ Screenshot::Screenshot(QOpenGLWidget *glwidget, QObject *parent)
 
 void Screenshot::takeScreenshot()
 {
-  screenshot_ = viewport_->grabFramebuffer();
+	screenshot_ = getScreenshot();
 }
 
-QImage Screenshot::getScreenshot() {
-  return viewport_->grabFramebuffer();
+QImage Screenshot::getScreenshot()
+{
+	QImage image = viewport_->grabFramebuffer();
+	for (int j = 0; j < image.height(); ++j)
+	{
+		QRgb *row = (QRgb*)image.scanLine(j);
+		for (int i = 0; i < image.width(); ++i)
+		{
+			((uint8_t*)&row[i])[3] = 255;
```

Comment:
Here is a good place for a comment, or a function call. May I suggest 
```suggestion
			fixAlphaChannel(row[i]);
```

Reply:
Or whichever pixel value you're fixing there...

Reply:
then above
```
  static constexpr int ALPHA_INDEX = 3;
  static constexpr int OPAQUE = 255;
  using Rgba8bit = uint8_t*;
static void fixAlphaChannel(QRgb& rgb)
{
  static_cast<Rgba8bit>(&rgb)[ALPHA_INDEX] = OPAQUE;
}
```

Reply:
And this doesn't change the mac/linux output at all? I'll run it on Windows today

Reply:
it theoretical shouldn't. It's just setting all pixels to be fully opaque. For some reason (I'm guessing a QT bug) non-opaque pixels cause strange behavior and the QT built-in conversion function that's supposed to throw away the alpha channel seems to use the same broken code and doesn't fix the problem.